public_html/wp-content/mu-plugins/blocks/blocks.php

```diff
@@ -0,0 +1,119 @@
+<?php
+namespace WordCamp\Blocks;
+defined( 'WPINC' ) || die();
+
+use WP_Post;
+
+define( __NAMESPACE__ . '\PLUGIN_DIR', \plugin_dir_path( __FILE__ ) );
+define( __NAMESPACE__ . '\PLUGIN_URL', \plugins_url( '/', __FILE__ ) );
+
+/**
+ * Load files.
+ *
+ * @return void
+ */
+function load() {
+	if ( ! function_exists( 'register_block_type' ) ) {
+		return;
+	}
+
+	require_once PLUGIN_DIR . 'includes/speakers.php';
+}
+
+add_action( 'plugins_loaded', __NAMESPACE__ . '\load' );
+
+/**
+ * Add block categories for custom blocks.
+ *
+ * @param array   $default_categories
+ * @param WP_Post $post
+ *
+ * @return array
+ */
+function register_block_categories( $default_categories, $post ) {
+	$default_categories[] = array(
+		'slug'  => 'wordcamp',
+		'title' => __( 'WordCamp Blocks', 'wordcamporg' ),
+	);
+
+	return $default_categories;
+}
+
+add_filter( 'block_categories', __NAMESPACE__ . '\register_block_categories', 10, 2 );
+
+/**
+ * Register assets.
+ *
+ * The assets get enqueued automatically by the registered block types.
+ *
+ * @return void
+ */
+function register_assets() {
+	wp_register_style(
+		'wordcamp-blocks',
+		PLUGIN_URL . 'assets/blocks.min.css',
+		[],
+		filemtime( PLUGIN_DIR . 'assets/blocks.min.css' )
```

Comment:
This is for wordcamp.org rather than being distributed in the plugin directory, so there isn't any formal version.

I think OPCache and WP Super Cache negate any performance impact. I've never seen or heard about it producing false positives, but that could be a concern; what have you seen happen?

The big benefit of `filemtime` is that it avoids the inconvenience and human error associated with manually bumping a number. It's very common for people to forget to do it, so the changes get deployed and but users don't see them unless they hard-refresh, and there can be conflicts between what the new version of 1 file expects and what the cached version of another file does.